eq-author-api/tests/utils/contextBuilder/comments/updateComment.js

```diff
@@ -0,0 +1,25 @@
+const executeQuery = require("../../executeQuery");
+
+const updateCommentMutation = `
+    mutation updateComment($input: UpdateCommentInput!) {
+  updateComment(input: $input) {
+      id
+      commentText
+      editedTime
+  }
+}
```

Comment:
Something off with the indendations in the graphql here.
Should be in this format:
```js
const updateCommentMutation = `
  mutation updateComment($input: UpdateCommentInput!) {
    updateComment(input: $input) {
      id
      commentText
      editedTime
    }
  }
`;
```
Seems like it is the same with the other files in this folder.